datastore/core/serialize.py

```diff
@@ -10,17 +11,12 @@ class Serializer(object):
     @classmethod
     def loads(cls, value):
         """returns deserialized `value`."""
-        raise NotImplementedError
+        raise NotImplementedError()
```

Comment:
*Note*: Here it's fine to raise however, because `.loads` is not a special Python method like `__lt__`

Reply:
Okay. Got it 👍 Thanks